webui/react/src/pages/ExperimentDetails/ExperimentVisualization.tsx

```diff
@@ -195,7 +195,8 @@ const ExperimentVisualization: React.FC<Props> = ({ basePath, experiment }: Prop
      * In the case of Custom Searchers, all the tabs besides
      * "Learning Curve" aren't helpful or relevant, so we are hiding them
      */
-    if (!(filters.maxTrial && filters.batchMargin && filters.batch)) return [];
+    if (!(filters.maxTrial && filters.batchMargin && (filters.batch || filters.batch === 0)))
```

Comment:
prob `filters.batch !== undefined` or `typeof filters.batch === 'number'` is more concise since we wanna show chart